### PR TITLE
Start using Cards in the Service details Page

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

**Comment by garrett** [comment]
Looks great!

I wonder if we should toss in the ct-form-info to the ct-form for the layout in the top card. (It also shouldn't be using `<label>`, but we can either fix that here or in another PR. It's not a regression; it's just wrong and has been for a while.)

**Review by garrett — APPROVED** [review]
:+1:

File: pkg/systemd/services/service-details.jsx [hunk 1]
```diff
@@ -126,9 +127,9 @@ export class ServiceTemplate extends React.Component {
 
     render() {
         return (
-            <>
-                {this.state.error && <Alert variant="danger" isInline title={this.state.error} />}
-                <div className="panel panel-default">
+            <Card>
+                <CardBody>
+                    {this.state.error && <Alert variant="danger" isInline title={this.state.error} />}
```
Comment (marusak) [inline pkg/systemd/services/service-details.jsx:132]: I am not sure if this alert should be in the card or outside of it.
on this pr:
incard
moving it outside of the card:
outsideofcard

Also there is that line, not sure where it comes from :man_shrugging:

**Review by marusak — APPROVED** [review]
I just have one (completely harmless) question. I am approving as it is likely what we actually want, just wanted to point it out. Otherwise looks good, thanks!

File: pkg/systemd/services/service-details.jsx (hunk 1 above)
Comment (KKoukiou) [reply pkg/systemd/services/service-details.jsx:132]: @garrett 
I tend to believe to Alert should be out, but I leave it up to you.
(I will check the line)

Comment (martinpitt) [reply pkg/systemd/services/service-details.jsx:132]: Not sure what you mean by "that line", but the strong red one on top of the alert box is part of the design: https://www.patternfly.org/v4/documentation/react/components/alert

Comment (KKoukiou) [reply pkg/systemd/services/service-details.jsx:132]: There is a thin line above the template name, inside the card, I just noticed as well.

Comment (garrett) [reply pkg/systemd/services/service-details.jsx:132]: This is a really weird page. There's a random `<hr>` for no reason, some space below the button, no heading, and no guidance at all what someone is supposed to do here. Additionally, I think the input is unstyled.

The error should either be inside like the first item (which feels weird to me) or outside the card (treating it as a complete unit) and not be styled like an inline alert, but as a default one.

Comment (garrett) [reply pkg/systemd/services/service-details.jsx:132]: I really don't understand what's supposed to be happening here at all. (And if I don't, then 98% of our users probably wouldn't either.)

Comment (garrett) [reply pkg/systemd/services/service-details.jsx:132]: Perhaps we should just leave it inline, in the card, and come back to this later. It already has all sorts of odd issues. We could do a follow-up to attempt this page.

Comment (KKoukiou) [reply pkg/systemd/services/service-details.jsx:132]: ack thanks, so I am merging it as is.

**Pull request merged**